protocol/contracts/v2/TreasuryPriceIndexOracle.sol

```diff
@@ -101,8 +101,10 @@ contract TreasuryPriceIndexOracle is ITreasuryPriceIndexOracle, TempleElevatedAc
         uint96 _oldTpi = tpiData.currentTpi;
         uint96 _newTpi = value.encodeUInt96();
 
-        uint256 _delta = (_newTpi > _oldTpi) ? _newTpi - _oldTpi : _oldTpi - _newTpi;
```

Comment:
FYI some extra optimisations were added to the issue #12 later, and added in https://github.com/TempleDAO/temple/pull/842/commits/8dd4a4a44447f69ada84dba8b7ad2fc19873c86d